client/src/components/home/footer.tsx

```diff
@@ -4,11 +4,11 @@ import Link from 'next/link';
 import { ExternalLink } from 'lucide-react';
 import EuropeFlag from 'public/images/europe-flag.svg';
```

Comment:
I'm only seeing one quarter of Europe here

Reply:
Nice catch! The issue seems to only be on Chrome 🕵️.